src/backend/gl/src/queue.rs

```diff
@@ -115,7 +115,7 @@ impl Queue {
                 (glow::BYTE, glow::SHORT, glow::INT),
             C::Uint | C::Unorm =>
                 (glow::UNSIGNED_BYTE, glow::UNSIGNED_SHORT, glow::UNSIGNED_INT),
-            C::Float => (glow::ZERO, glow::HALF_FLOAT, glow::FLOAT),
```

Comment:
Accidentally removed?

Reply:
Yes, though the commented out code shouldn't really be here in the first place. We have git for that.

Reply:
Maybe it would be better to remove all commented code?